### PR TITLE
[FIRRTL] Remove the FIRRTL parameter attributes

### PR DESCRIPTION
**Pull request by rwy7**
Just use the HW dialect's parameters directly, instead.

### CODE REVIEW

File: include/circt/Dialect/FIRRTL/FIRRTLStructure.td [hunk 1]
```diff
@@ -86,7 +87,6 @@ class FIRRTLModuleLike<string mnemonic, list<Trait> traits = []> :
 
 }
 
-
```
Comment (seldridge) [inline include/circt/Dialect/FIRRTL/FIRRTLStructure.td:89]: Mega-nit: feel free to batch whitespace changes into NFC commits when appropriate.

**Review by seldridge — APPROVED** [review]
LGTM

I agree with trying to reuse this and see where we get.  I can't recall my reasoning for not doing this in the first place. 🤔  Thanks for cleaning this up!

**Review by dtzSiFive — APPROVED** [review]
LGTM, thanks! Do you know, were there any differences between the two at all (besides FIRRTL vs HW? :laughing:)?

**Comment by darthscsi** [comment]
Status?  If we want to do it, let's close out on it.  One concern is whether firrtl will have the same attribute operation tree as HW.  If not, then we probably want it's own attribute.

**Pull request closed without merging**